alembic/versions/2020032904_add_tag_exclusive_workers_9d0f1ffb18e9.py

```diff
@@ -0,0 +1,22 @@
+"""Add tag_exclusive workers
+
+Revision ID: 9d0f1ffb18e9
+Revises: 75d4288ae265
+Create Date: 2020-03-29 04:57:59.567422
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '9d0f1ffb18e9'
+down_revision = '75d4288ae265'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('worker', sa.Column('tag_exclusive', sa.Boolean(), nullable=False, server_default='0'))
```

Comment:
I found we have several places in our codebase have inconsistency between database definition and the real value that we are retrieving/setting. For example, we are setting the new column `tag_exclusive` with the default value `0` in the database. However, when we process this field later in either worker side or bundle-manager, we treat it as `True` or `False`. In the code logic, it doesn't seem to have a huge impact on the functionality. I just think it might be a bit more clear to make them consistent so that we don't need to guess around what the real value is.

Reply:
Do you want me to address this in this PR, or is this a separate issue?

Reply:
Could we change the `server_default` value to be `True` or `False`? Not sure if this is the right syntax though.